app/experimenter/experiments/management/commands/generate_dashboards.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from django.conf import settings
+
+from redash_client.client import RedashClient
+from redash_client.dashboards.StatisticalDashboard import (
+    StatisticalDashboard)
+
+from django.core.management.base import BaseCommand
+from experimenter.experiments.models import Experiment
+
+
+class Command(BaseCommand):
+    POPULATION_TEMPLATE = 'UT Experiment Template: Population Size'
+    EVENTS_PER_HOUR_TEMPLATE = 'TTests Template Per Hour UT Five:'
+    UT_HOURLY_TTABLE = "Statistical Analysis (Per Active Hour) - UT"
+    help = 'Generates Redash dashboards'
+
+    def generate_dashboards(self):
+        redash_client = RedashClient(settings.REDASH_API_KEY)
+
+        relevant_experiments = Experiment.objects.filter(
+            status__in=(
+                Experiment.STATUS_LAUNCHED,
+                Experiment.STATUS_COMPLETE))
+        for exp in relevant_experiments:
+            try:
+                dash = StatisticalDashboard(
+                  redash_client,
+                  settings.AWS_ACCESS_KEY,
+                  settings.AWS_SECRET_KEY,
+                  settings.S3_BUCKET_ID_STATS,
+                  exp.project.name,
+                  exp.name,
+                  exp.slug,
+                  start_date=exp.start_date.strftime("%Y-%m-%d")
+                )
+                dash.add_graph_templates(self.POPULATION_TEMPLATE)
+                dash.add_ttable_data(
+                  self.EVENTS_PER_HOUR_TEMPLATE,
+                  self.UT_HOURLY_TTABLE,
+                  dash.UT_HOURLY_EVENTS
+                )
+                dash.add_ttable(self.UT_HOURLY_TTABLE)
+
+                exp.dashboard_url = dash.public_url
+                exp.save()
+            except:
```

Comment:
This is called 'Pokemon exception handling' becuase it 'catches them all' 

You should explicitly list which types of exceptions are expected here, and anything that's not expected can hard crash the thing (this is the desired behaviour).